ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -245,6 +246,34 @@ - (void)stepDidChange {
     
 }
 
+- (NSAttributedString *)getAttributedQuestionTitle {
+    NSString *openFont = @"<font size='6' style='font-family:HelveticaNeue-Light'>";
+    NSString *closeFont = @"</font>";
+    NSString *customTitle = [NSString stringWithFormat:@"%@%@%@", openFont, self.questionStep.title, closeFont];
+    NSError *error = nil;
+    NSMutableAttributedString *attts = [[NSMutableAttributedString alloc] initWithData:[customTitle dataUsingEncoding:NSUnicodeStringEncoding] options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:nil error:&error];
+    NSRange range = NSMakeRange(0, attts.length);
+    NSMutableParagraphStyle *style =  [[NSMutableParagraphStyle alloc] init];
+    style.alignment = NSTextAlignmentCenter;
+    [attts addAttribute:NSParagraphStyleAttributeName value:style range:range];
+    
+    return attts;
+}
+
+- (NSAttributedString *)getAttributedQuestionText {
+    NSString *openFont = @"<font size='4' style='font-family:HelveticaNeue-Light'>";
+    NSString *closeFont = @"</font>";
+    NSString *customTitle = [NSString stringWithFormat:@"%@%@%@", openFont, self.questionStep.text, closeFont];
+    NSError *error = nil;
+    NSMutableAttributedString *attts = [[NSMutableAttributedString alloc] initWithData:[customTitle dataUsingEncoding:NSUnicodeStringEncoding] options:@{NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType} documentAttributes:nil error:&error];
+    NSRange range = NSMakeRange(0, attts.length);
+    NSMutableParagraphStyle *style =  [[NSMutableParagraphStyle alloc] init];
+    style.alignment = NSTextAlignmentCenter;
+    [attts addAttribute:NSParagraphStyleAttributeName value:style range:range];
+    
+    return attts;
+}
```

Comment:
ok, probably the same for all of these

Reply:
sure, I centralised a method in ORKStepHeaderView file